postBuildStep.ts

```diff
@@ -38,27 +38,25 @@ async function copyStaticFiles() {
     await fse.ensureDir(commonOutDir);
     await copyFile(commonSrcDir, commonOutDir, "styles.css");
 
-    // Must set environment variables EDGE_CHROMIUM_PATH and EDGE_CHROMIUM_OUT_DIR
-    // E.g. set EDGE_CHROMIUM_PATH=F:/git/Edge/src
-    //      set EDGE_CHROMIUM_OUT_DIR=Release
-    // See CONTRIBUTING.md for more details
+    const dirName = __dirname.replace(/\\/g, '/');
+    const sourceFilesPath = dirName + '/out/edge/src';

```

Comment:
lets use path to normalize dirName and to join dirName with out/edge/src. That way we don't neet to worry about \ vs / on linux, mac or windows.

Reply:
Fixed.  Didn't realize that the path library was perfect for this kind of thing.  Thanks!